index.bs

```diff
@@ -10574,14 +10574,20 @@ Operations that happen <dfn>atomically</dfn> on a
 given thread can only be executed when no other [=Atomically|atomic=]
 operation is running on another thread.
 
-The algorithm for rendering a block of audio from an
-{{AudioContext}} <em>G</em> with a <a href="#control-message-queue">control message queue</a> <em>Q</em> is as
-follows.
-
-If the algorithm returns true then it MUST be executed again in the
-future, to render the next block of audio. Else, the <a>rendering
-thread</a> yields and the processing stops. The <a>control thread</a>
-can restart executing this algoritm if needed.
+The algorithm for rendering a block of audio from an {{AudioContext}} <em>G</em>
+with a <a href="#control-message-queue">control message queue</a> <em>Q</em> is
+comprised of multiple steps and the order of these steps cannot be altered or
+interrupted:
+	1. Prepare the rendering.
+	2. Process the control message queue.
+	3. Process a render quantum.
+	4. Perform a microtask check point.
+	5. Return the result of the rendering.
+
+If the step 5 returns true then it MUST be executed again in the future, to
+render the subsequent block of audio. Otherwise, the <a>rendering thread</a>
+yields and the processing stops. The <a>control thread</a> can restart executing
+these steps if needed.
```

Comment:
If you do link to the algorithm, then this paragraph doesn't really provide an additional detail and isn't needed, right?

Reply:
Done.